R/adjective.R

```diff
@@ -88,6 +88,7 @@ adjective <- c(
   "primo",
   "proper",
   "rad",
+  "rockandroll",
```

Comment:
:) Thanks! Could you please move it after `riveting`, so that it is in alphabetical order.